packages/web-pkg/src/composables/piniaStores/app.ts

```diff
@@ -0,0 +1,16 @@
+import { defineStore } from 'pinia'
+import { ref } from 'vue'
+
+export interface AppError {
+  message: string
+  status?: number
+}
+
+export const useAppStore = defineStore('app', () => {
```

Comment:
Just out of curiosity: why did you use store for this instead of e.g. event from the app like is done with the `update:resource` event?